pkg/config/config.go

```diff
@@ -137,6 +148,7 @@ func ReadFromCommandLineFlags() {
 		"The path to the config file (i.e poseidon_cfg) without filename or extension, supported extensions/formats are Yaml, Json")
 	flag.BoolVar(&config.EnablePprof, "enablePprof", false, "Enable runtime profiling data via HTTP server. Address is at client URL + \"/debug/pprof/\"")
 	flag.StringVar(&config.PprofAddress, "pprofAddress", "0.0.0.0:8989", "Address on which to collect runtime profiling data,default to set for all interfaces ")
+	pflag.StringVar(&config.MetricsBindAddress, "metricsBindAddress", "0.0.0.0:8989", "The port that the poseidon's http service runs on,default to set for all interfaces")
```

Comment:
parameter message does not match the parameter. It should be something like,

Address on which to serve poseidon runtime metrics.

Reply:
eeem,  good point